tests/gem5/kvm_switch_tests/README.md

```diff
@@ -0,0 +1,8 @@
+# KVM Switch Tests
+
+These tests ensure that gem5 can switch processors during simulation.
+To run these tests by themselves, you can run the following command in the tests directory:
+
```

Comment:
Similar to above, in another PR, it would be good to specify the system requirements to run these tests.

Reply:
Sounds good, I can make the extra changes to the READMEs after this PR gets in